### DISCUSSION TITLE
winform should provide official ribbon controls

### DISCUSSION THREAD

**Discussion by ygc369** [dotnet/winforms#11203] [Ideas] [answered]
3rd-party DevExpress has winform ribbon controls, but it is not free. We hope winform can provide official ribbon controls.

**Comment by harborsiem** [comment, marked as answer]
Here you can have a look to my open source project [WindowsRibbon](https://github.com/harborsiem/WindowsRibbon). This is a C# wrapper to the Windows Ribbon Framework (UIRibbon.dll).
[Nuget package](https://www.nuget.org/packages/windowsribbon).

**Reply by tbolon** [reply]
Added [on my list](https://github.com/tbolon/awesome-dotnet-winforms/blob/main/README.md#menu--navigation--dialogs--tabs--windows) ;)
